monitoring/grafana/dashboards/tests/features/hosts_overview.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Hosts Overview Dashboard
+
+Scenario: "Test network load succeeds"
+  Given the following series:
+    | metrics | values |
+    | node_network_receive_bytes{instance="127.0.0.1", device="eth1"} | 10 100 |
+    | node_network_receive_bytes{instance="127.0.0.1", device="eth2"} | 10 100 |
+    | node_network_transmit_bytes{instance="127.0.0.1", device="eth1"} | 10 100 |
+    | node_network_transmit_bytes{instance="127.0.0.1", device="eth2"} | 10 100 |
+  When variable `osd_hosts` is `127.0.0.1`
```

Comment:
Is this expansion something a user should really know about? Could we simply hide this and pre-fill all template variables with a default, and ONLY if it makes sense for a test to customize this, then we allow the user to specify it?

Reply:
I prefilled all variables with an `UNSET VARIBLE` string so it's easy to spot.

Reply:
Good!